td-layout/src/memslice.rs

```diff
@@ -55,6 +57,10 @@ pub fn get_mem_slice<'a>(t: SliceType) -> &'a [u8] {
             SliceType::TdHob => {
                 core::slice::from_raw_parts(TD_HOB_BASE as *const u8, TD_HOB_SIZE as usize)
             }
+            SliceType::KernelParameter => core::slice::from_raw_parts(
+                KERNEL_PARAM_BASE as *const u8,
+                KERNEL_PARAM_SIZE as usize,
```

Comment:
Presumably if the params occupy < `KERNEL_PARAM_SIZE` bytes, the rest of the space is guaranteed to be zero'd? Is it worth adding a test for this since it seems like the length could vary depending on workload?

Reply:
VMM put the params into memory and td shim doesn't  known the actual params occupy, so we can't determine the size of memory should be zeroed here.